[AUTH-133] Add pod security admission control docs (#47647)
openshift/openshift-docs: authentication/understanding-and-managing-pod-security-admission.adoc

:_content-type: ASSEMBLY
[id="understanding-and-managing-pod-security-admission"]
= Understanding and managing pod security admission
include::_attributes/common-attributes.adoc[]
:context: understanding-and-managing-pod-security-admission

> toc::[]

Author: I'm immediately outsourcing that to the K8s docs. As a lede, though, see other comment.

> {product-title} supports link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[pod security admission]. Clusters monitor administrator-labeled namespaces for changes and map link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc[security context constraints].

Author: ```suggestion
{product-title} supports link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[pod security admission]. Clusters monitor administrator-labeled namespaces for changes and map link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc#security-context-constraints-about_configuring-internal-oauth[security context constraints].
```

> {product-title} supports link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[pod security admission]. Clusters monitor administrator-labeled namespaces for changes and map link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc#security-context-constraints-about_configuring-internal-oauth[security context constraints].

Author: ```suggestion
Pod security admission is an implementation of the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[Kubernetes pod security standards]. Use pod security admission to restrict the behavior of pods.
```

> You can learn more about admission controller configuration in the link:https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/#configure-the-admission-controller[Kubernetes documentation].

Author: ```suggestion
```
